pubspec.yaml

```diff
@@ -20,3 +20,9 @@ dev_dependencies:
   build_runner: ^2.1.4
   lints: ^1.0.1
   coverage: ^1.0.3
+
+dependency_overrides:
```

Comment:
Deploying nyxx 4.0.0-dev.0 as a prerelease and using that as the target version would be cleaner

Reply:
probably yes but its WIP so thats not an issue for know. When feature will be available for everyone will push dev version for preview